### PR TITLE
Ryna - LB161.ag_R_C_Y_GLU_irr

### PR DESCRIPTION
**Pull request by rynacui**
#646

### CODE REVIEW

File: R/zchunk_L1211.nonco2_awb_R_S_T_Y_IRR.R [hunk 1]
```diff
@@ -19,8 +19,8 @@
 #' @author CDL April 2017
 module_emissions_L1211.nonco2_awb_R_S_T_Y_IRR <- function(command, ...) {
   if(command == driver.DECLARE_INPUTS) {
-    return(c(FILE = "temp-data-inject/L161.ag_irrProd_Mt_R_C_Y_GLU",
-             FILE = "temp-data-inject/L161.ag_rfdProd_Mt_R_C_Y_GLU",
```
Comment (bpbond) [inline R/zchunk_L1211.nonco2_awb_R_S_T_Y_IRR.R]: Did you delete these files out of `temp-data-inject/`?

Comment (rynacui) [inline R/zchunk_L1211.nonco2_awb_R_S_T_Y_IRR.R]: Yes, I did.

**Comment by bpbond** [comment]
Holding on merging this until we resolve #656

**Comment by bpbond** [comment]
Runs fine on my system, so merging even though Travis is borked ( #656 ).

**Pull request merged**